### ISSUE TITLE
shelf_router_generator: upgrade dependencies

### ISSUE THREAD

**Issue by isoos** [google/dart-neats#4]
Current constraints block its use on pub site:
````
Because every version of shelf_router_generator depends on analyzer ^0.35.0 and pana 0.12.16 depends on analyzer ^0.36.0, shelf_router_generator is incompatible with pana 0.12.16.
So, because pub_dartlang_org depends on both pana 0.12.16 and shelf_router_generator ^0.7.0+1, version solving failed.
````

**Comment by jonasfj** [comment]
We use `analyzer` for codegen, we should simply bump the version constraint...

**Comment by jonasfj** [comment]
Fixed in `0.7.1`
